synthtool/transforms.py

```diff
@@ -80,7 +80,7 @@ def _copy_dir_to_existing_dir(
         excludes = []
     for root, _, files in os.walk(source):
         for name in files:
-            rel_path = str(Path(root).relative_to(source)).lstrip(".")
```

Comment:
this was causing issues with hidden dirs/files. I didn't see adverse effects from removing this, but @theacodes maybe you recall why we did this?